shipit.yml

```diff
@@ -6,8 +6,9 @@ review:
 
 deploy:
   override:
-    - make install:
-        timeout: 5000
+    - |
+      bash deploy.sh
```

Comment:
Is this deploy.sh still necessary? The shell script checks if branch=giraffe-develop, which won't be used anymore & therefore won't deploy at all. I think maybe we should change our environments in our stacks in shipit to be registration_<environment> /  reporting_<environment> and make specific shipit.<app>_<environment>.yaml files.

Reply:
We can probably revert this yes

Reply:
We might be able to do something with the shipit stack environments here to coordinate which FE app we want to deploy